Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -5843,3 +5844,26 @@ webkit_web_view_get_default_content_security_policy(WebKitWebView* webView)
 
     return webView->priv->defaultContentSecurityPolicy.data();
 }
+
+/**
+ * webkit_web_view_end_immersive_session:
+ * @web_view: a #WebKitWebView
+ *
+ * Requests the immersive session associated to this #WebKitWebView to end.
```

Comment:
We need to at least explain what an immersive session is. (And also how to detect that one has started, when adding the property that Carlos requested.)